locations/spiders/pick_n_pay.py

```diff
@@ -72,7 +96,31 @@ def parse(self, response, **kwargs):
                         "extras": Categories.SHOP_DOITYOURSELF.value,
                     }
                 )
-            # Unhandled: "", "BPFORECOURT", "DAILY", "DC", "FAMILY", "HYPER", "LOCAL", "MARKET",
-            # "MINI", "NO_FORMAT", "ONLINE", "REGIONAL OFFICE", "SUPER", "WHOLESALE"
+            elif store["storeType"] == "DC":  # Distribution Centre
+                item.update(
+                    {
+                        "extras": Categories.INDUSTRIAL_WAREHOUSE.value,
```

Comment:
`industrial=warehouse` isn't really used anywhere in OSM. Maybe `building=warehouse`?

Reply:
[Over 7000 uses](https://wiki.openstreetmap.org/wiki/Key:industrial) is something, and I don't really consider `building=warehouse` as a feature tag

Reply:
I didn't see it on taginfo at first glance, but I do now. Apologies!